### PR TITLE
Fix a circular dependency on vehLight packet

### LINKED ISSUE #1915: Uncaught Exception: Converting circular structure to JSON
Uncaught Exception: Converting circular structure to JSON
    --> starting at object with constructor 'Vehicle2016'
    |     property 'positionUpdate' -> object with constructor 'Object'
    --- property 'vehicle' closes the circle time : Tue Feb 06 2024 23:50:36 GMT+0000 (Coordinated Universal Time)
TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Vehicle2016'
    |     property 'positionUpdate' -> object with constructor 'Object'
    --- property 'vehicle' closes the circle
    at JSON.stringify (<anonymous>)
    at H1Z1Protocol.pack (/usr/src/app/node_modules/h1z1-server/out/protocols/h1z1protocol.js:311:59)
    at LiveZoneServer._sendData (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:2552:37)
    at LiveZoneServer.sendData (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:2564:14)
    at LiveZoneServer.vehicleManager (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:3085:26)
    at LiveZoneServer.executeRoutine (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zoneserver.js:5411:14)
    at Timeout._onTimeout (/usr/src/app/node_modules/h1z1-server/out/servers/ZoneServer2016/zonepackethandlers.js:395:24)
    at listOnTimeout (node:internal/timers:573:17)
    at process.processTimers (node:internal/timers:514:7)

### PR DESCRIPTION
**Pull request by QuentinGruber**
---

<details open="true"><summary>Generated summary (powered by <a href="https://app.graphite.dev">Graphite</a>)</summary>

> ## TL;DR
> This pull request addresses issue #1915 by ensuring that the `vehicle` property in the `positionUpdate` object is set to `undefined` in both the `Plane` and `Vehicle2016` classes.
> 
> ## What changed
> Changes were made in two files: `plane.ts` and `vehicle.ts`. In both files, the `vehicle` property in the `positionUpdate` object was set to `undefined`. This was done to prevent any potential issues or conflicts that could arise from this property being defined in these classes.
> 
> ```diff
> -        ...this.positionUpdate
> +        ...this.positionUpdate,
> +        vehicle: undefined // To fix #1915
> ```
> 
> ## How to test
> To test these changes, follow these steps:
> 
> 1. Pull the changes from this branch into your local environment.
> 2. Run the server and ensure that it starts without any errors.
> 3. Interact with a `Plane` or `Vehicle2016` entity and observe the `positionUpdate` object. The `vehicle` property should be `undefined`.
> 4. Check the server logs to ensure that no errors or warnings are being thrown related to these changes.
> 
> ## Why make this change
> This change was made to address issue #1915. By setting the `vehicle` property to `undefined`, we can prevent potential issues or conflicts that could arise from this property being defined in the `Plane` and `Vehicle2016` classes. This makes our code more robust and less prone to bugs in the future.
</details>

### CODE REVIEW

**Comment by QuentinGruber** [comment]
Current dependencies on/for this PR:
* `dev`
  * **PR #1935** <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1935?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1935?utm_source=stack-comment).
